docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md

```diff
@@ -106,7 +106,7 @@ Save your credentials in a safe place. If you lose these credentials you will no
 
 ## Viewing the cluster
 
-After you complete the wizard, the [**Clusters**](../../cloud-clusters/) page appears with the provisioning of your new cluster in progress. 
+After you complete the wizard, the [**Clusters**](../../cloud-clusters/) page appears with the provisioning of your new cluster in progress.
 
 ![Cluster being provisioned](/images/yb-cloud/cloud-cluster-provisioning.png)
 
```

Comment:
line 113 suggestion: Once the cluster is ready, the [cluster overview](../../cloud-clusters/overview) is displayed OR Once the cluster is ready, the cluster [overview](../../cloud-clusters/overview) is displayed.